editor.planx.uk/src/pages/FlowEditor/components/forms/FormModal.tsx

```diff
@@ -62,6 +62,7 @@ const NodeTypeSelect: React.FC<{
         <option value={TYPES.AddressInput}>Address Input</option>
         <option value={TYPES.ContactInput}>Contact Input</option>
         <option value={TYPES.List}>List</option>
+        <option value={TYPES.MapAndLabel}>Map and Label</option>
```

Comment:
nit: I'd comment-out this line in this initial PR - basically like a pseudo feature flag so no one can select/use this component type yet ! Or create a new "TREES" feature flag and wrap it :slightly_smiling_face: :deciduous_tree: